### PR TITLE
changed parameter names from switch to skip

### PR DESCRIPTION
**Pull request by Lucpen**
## PR checklist

- [x] changed parameter names from switch to skip to make their name more intuitive
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/genomic-medicine-sweden/tomte/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: nextflow_schema.json [hunk 1]
```diff
@@ -260,34 +260,34 @@
                     "enum": ["consensus", "multiallelic"],
                     "help_text": "Bcftools call can eitherbe run in multiallelic mode or in consensus mode. In consensus mode a p-value threshold of 0.01 is applied."
                 },
-                "switch_build_tracks": {
+                "skip_build_tracks": {
                     "type": "boolean",
-                    "default": true,
-                    "description": "Should tracks be build?",
+                    "default": false,
+                    "description": "Should track building be skipped?",
```
Comment (jemten) [inline nextflow_schema.json:266]:
```suggestion
                    "description": "Skip building splice junction tracks for IGV.",
```

File: nextflow_schema.json [hunk 2]
```diff
@@ -260,34 +260,34 @@
...
+                    "description": "Should VEP module be skipped?",
                     "fa_icon": "fas fa-toggle-off"
                 },
-                "switch_drop_ae": {
+                "skip_drop_ae": {
                     "type": "boolean",
-                    "default": true,
-                    "description": "Should DROP Aberrant Expression module be run?",
+                    "default": false,
+                    "description": "Should DROP Aberrant Expression module be skipped?",
```
Comment (jemten) [inline nextflow_schema.json:284]:
```suggestion
                    "description": "Skip DROP Aberrant Expression module ",
```

File: nextflow_schema.json [hunk 3]
```diff
@@ -260,34 +260,34 @@
...
                     "fa_icon": "fas fa-toggle-off"
                 },
-                "switch_drop_as": {
+                "skip_drop_as": {
                     "type": "boolean",
-                    "default": true,
-                    "description": "Should DROP Aberrant Splicing module be run?",
+                    "default": false,
+                    "description": "Should DROP Aberrant Splicing module be skipped?",
```
Comment (jemten) [inline nextflow_schema.json:290]:
```suggestion
                    "description": "Skip DROP Aberrant Splicing module",
```

**Review by jemten — APPROVED** [review]
Looks good! I mainly had some suggestions for the help text

File: docs/usage.md [hunk 4]
```diff
@@ -123,7 +123,7 @@ If you would like to see more examples of what a typical samplesheet looks like
...
+Note that the pipeline is modular in architecture. It offers you the flexibility to choose between different tools. For example, you can call SNVs either with BCFtools or with GATK. You also have the option to turn off sections of the pipeline if you do not want to run them. For example, drop aberrant expression module can be turned off by setting `--skip_drop_ae TRUE`. This flexibility means that in any given analysis run, a combination of tools included in the pipeline will not be executed. So the pipeline is written in a way that can account for these differences while working with reference parameters. If a tool is not going to be executed during the course of a run, parameters used only by that tool need not be provided. For example, if you are not running DROP aberrant splicing, you do not need to provide `--reference_drop_splice_folder`.
```
Comment (Lucpen) [inline docs/usage.md:126]:
```suggestion
Note that the pipeline is modular in architecture. It offers you the flexibility to choose between different tools. For example, you can call SNVs either with BCFtools or with GATK. You also have the option to turn off sections of the pipeline if you do not want to run them. For example, drop aberrant expression module can be turned off by setting `--skip_drop_ae true`. This flexibility means that in any given analysis run, a combination of tools included in the pipeline will not be executed. So the pipeline is written in a way that can account for these differences while working with reference parameters. If a tool is not going to be executed during the course of a run, parameters used only by that tool need not be provided. For example, if you are not running DROP aberrant splicing, you do not need to provide `--reference_drop_splice_folder`.
```

File: nextflow_schema.json [hunk 5]
```diff
@@ -205,16 +205,16 @@
                     "enum": ["Basic", "None"],
                     "fa_icon": "fas fa-tachometer-alt"
                 },
-                "switch_subsample_region": {
+                "skip_subsample_region": {
                     "type": "boolean",
-                    "default": true,
-                    "description": "Do you want to subsample the region subsample_bed to the fraction given in seed_frac?",
+                    "default": false,
+                    "description": "Should subsampling the region subsample_bed to the fraction given in seed_frac be skipped?",
```
Comment (Lucpen) [inline nextflow_schema.json:211]:
```suggestion
                    "description": "Turn off subsampling of the region. The region is defined by the subsample_bed parameter and the fraction is given by seed_frac",
```

File: docs/usage.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (Lucpen) [reply docs/usage.md]: I am unsure, so I substituted FALSE for false

**Pull request merged**